homeassistant/components/sems_portal/sems_login.py

```diff
@@ -0,0 +1,20 @@
+"""Used for login logic."""
+
+from typing import Any
+
+from aiohttp import ClientSession
+
+
+async def login_to_sems(session: ClientSession, account: str, pwd: str) -> Any:
```

Comment:
needs to move to external PyPi package (https://developers.home-assistant.io/docs/development_checklist/)